app/(pages)/things-every-employee-should-know/crypto-wallets/page.tsx

```diff
@@ -0,0 +1,32 @@
+"use client";
+import { motion } from "framer-motion";
+import { Typography } from "@/components/ui/typography";
+import Link from "next/link";
+
+export default function GitBasics() {
```

Comment:
the default function name should correspond to the page name. "CryptoWallets" in this case.